bundle/metadata/annotations.yaml

```diff
@@ -15,4 +15,4 @@ annotations:
   operators.operatorframework.io.test.config.v1: tests/scorecard/
 
   # Annotations to specify supported OCP versions.
-  com.redhat.openshift.versions: "v4.16-v4.17"
+  com.redhat.openshift.versions: v4.15-v4.16
```

Comment:
do we want this change?

Reply:
No. Getting rid of the quotes was a consistency thing brought up on the patch PR, and I didn't realize the version changed. Fixed.